training/distributed_training/pytorch/model_parallel/GPT2_huggingface_trainer/submit_smp_trainer.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        pp_degree = 2
</pre><p>Do you need pp degree 2 to fit a 1.5B model? We will try to avoid pipeline parallel as much as possible since it requires cross-node communication. But once if we want to enable it, we will also set microbatch &gt; 1, to enable the pipeline.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3577/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Ok, will check to see if another config works.</p>